Gemfile.lock

```diff
@@ -405,69 +409,68 @@ GEM
       terminal-table (>= 1.4.0)
       thor (>= 0.16.0)
     jmespath (1.6.2)
-    json (2.6.3)
-    jsonpath (1.1.2)
+    json (2.7.1)
+    jsonpath (1.1.5)
       multi_json
     jwt (2.7.1)
     kubeclient (4.11.0)
       http (>= 3.0, < 6.0)
       jsonpath (~> 1.0)
       recursive-open-struct (~> 1.1, >= 1.1.1)
       rest-client (~> 2.0)
-    large_object_store (1.3.4)
-    llhttp-ffi (0.4.0)
+    large_object_store (1.7.0)
+      zstd-ruby (~> 1.5.5)
+    llhttp-ffi (0.5.0)
       ffi-compiler (~> 1.0)
       rake (~> 13.0)
-    lograge (0.12.0)
+    lograge (0.14.0)
```

Comment:
### [lograge](https://github.com/roidrage/lograge)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 0.12.0 → 0.14.0

[_[change-log](https://github.com/roidrage/lograge/blob/master/CHANGELOG.md), ~~source-code~~, [gem-diff](https://my.diffend.io/gems/lograge/0.12.0/0.14.0)_]





<details>
<summary>Commits</summary>

A change of **20** commits. See the full changes on [the compare page](https://github.com/roidrage/lograge/compare/v0.12.0...v0.14.0).

These are the first 10 commits:
- (85daba1) [Update README.md](https://github.com/roidrage/lograge/commit/85daba1404b24fae1c4470f1c0db4352481120e3)
- (7795e82) [Bump github/codeql-action from 1 to 2 (#350)](https://github.com/roidrage/lograge/commit/7795e825d2155f24df6067e2b9d91dfbe3f9232e)
- (8e99845) [Add Formater to serialize Deep hash to KeyValue (#282)](https://github.com/roidrage/lograge/commit/8e998454836a413280475da772c3b7fd9f1832f0)
- (3e80fcf) [Fix readme typo (#351)](https://github.com/roidrage/lograge/commit/3e80fcf21a5253f46307aab2a728fad722941c7d)
- (da59a2d) [Set TargetRubyVersion (#352)](https://github.com/roidrage/lograge/commit/da59a2dcfef2ee9b8da741c7bbdcdda40c355ceb)
- (c737a13) [Add Rails 6 memory allocations to default log (with fixed te…](https://github.com/roidrage/lograge/commit/c737a13779fef5c2b6050ce9356fd7acb86a8905)
- (49267f9) [[CI] Test against rails 6.1 (#332)](https://github.com/roidrage/lograge/commit/49267f9bdc58192302dc4ccd6908d3460f90b544)
- (0395ce4) [Create FUNDING.yml](https://github.com/roidrage/lograge/commit/0395ce412ddad559dbec44cb2b3756aed32550d7)
- (397d7f4) [Remove extraneous `puts` from spec](https://github.com/roidrage/lograge/commit/397d7f43a8e83b12161258e6aca0f12e7a6c7f84)
- (f36f8d8) [Merge pull request #361 from carlosantoniodasilva/patch-1](https://github.com/roidrage/lograge/commit/f36f8d844275fa703c90ca8d172304cad0fe052b)


</details>